crates/wascc-host/src/oci.rs

```diff
@@ -5,7 +5,10 @@ use std::str::FromStr;
 pub(crate) const OCI_VAR_USER: &str = "OCI_REGISTRY_USER";
 pub(crate) const OCI_VAR_PASSWORD: &str = "OCI_REGISTRY_PASSWORD";
 
-pub(crate) async fn fetch_oci_bytes(img: &str) -> Result<Vec<u8>> {
+pub(crate) async fn fetch_oci_bytes(img: &str, allow_latest: bool) -> Result<Vec<u8>> {
+    if !allow_latest && img.ends_with(":latest") {
+        return Err("Fetching mutable images tagged 'latest' is prohibited".into());
```

Comment:
Do you see value in including in the error something along the lines of `you can override this with the --allow-latest flag`?

Reply:
Good point. I pushed a change to this, though I don't want to mention `--allow-latest` within the wascc-host code because it doesn't know which binary instantiated it.